mysql-test/suite/sys_vars/t/innodb_undo_log_encrypt_basic.test

```diff
@@ -0,0 +1,65 @@
+--source include/have_innodb.inc
+
+--disable_query_log
+call mtr.add_suppression("\\[ERROR\\] InnoDB: Encryption can't find master key, please check the keyring plugin is loaded.");
```

Comment:
Later: suppressions are already substring matches, easier to read and maintain if dropped those common "[ERROR] InnoDB:" prefixes